### DISCUSSION TITLE
How do I embed the widget in my form?

### DISCUSSION THREAD

**Discussion by sudhz** [mCaptcha/mCaptcha#158] [Q&A] [answered]
I read the docs and tried to look into the code, but couldn't figure out how would I embed the mCaptcha form inside of own JSP and Java Servlets project. It's kind of an old project, so any help would be appreciated. Thanks in advance!

**Comment by realaravinth** [comment, marked as answer]
Sorry, I'm not familiar with JSP. The HTML code to embed the widget is over [here](https://mcaptcha.org/docs/webmasters/installing-captcha#4-install-mcaptcha-on-your-website). Maybe someone familiar with JSP can help translate it :)

**Comment by sudhz** [comment]
The link that you've provided is enough to get started, thanks for the help!
